### PR TITLE
fix issue #228 replace start to run

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock 21521209@zju.edu.cn

### CODE REVIEW

**Comment by Mashimiao** [comment]
This has been talked in #184.  I think you'd better separate it into create and start.

**Comment by heartlock** [comment]
@Mashimiao good idea, I will update it. but I think when the work of issuse #66 is complete, test_runtime.sh will be removed. I don't know if  it is necessary to update it here.

**Comment by Mashimiao** [comment]
I think it is, as test_runtime.sh is the only runtime test tool by now.

**Comment by Mashimiao** [comment]
And there is no need two commits, just one commit will be fine.

File: test_runtime.sh [hunk 1]
```diff
@@ -79,8 +79,17 @@ cp runtimetest ${TESTDIR}
...
+conID=$(uuidgen)
+
+CREATECMD="${RUNTIME} create ${conID}"
+TESTCMD="${RUNTIME} start ${conID}"
 pushd $TESTDIR > /dev/null
+if ! ${CREATECMD}; then
+	info "Runtime is ${RUNTIME}, no need to create before start"
+else
+	info "Runtime is ${RUNTIME}, need to create before start"
+fi
```
Comment (heartlock) [inline test_runtime.sh]: when exitcode of command "${RUNTIME} create ${conID}" is 1, runtime has no command "create" continue to run command "start". If exitcode of command "${RUNTIME} create ${conID}" is -1, runtime create container fail and print error. but I don't know how to get exitcode of command "${RUNTIME} create ${conID}".

Comment (Mashimiao) [inline test_runtime.sh]: when executed "${RUNTIME} create ${conID}", you can use `$?` to get the exit code.
just as:
`# sleep 1
# echo $?
0`

Comment (heartlock) [inline test_runtime.sh]: but if runtime has no command "create", execute  "${RUNTIME} create ${conID}" will cause a crash.https://github.com/opencontainers/runtime-tools/blob/master/test_runtime.sh#L2

Comment (Mashimiao) [inline test_runtime.sh]: If necessary, I think we can remove it.

Comment (Mashimiao) [inline test_runtime.sh]: If necessary, I think we can remove it.

Comment (heartlock) [inline test_runtime.sh]: I think it is unwise to remove it.

Comment (Mashimiao) [inline test_runtime.sh]: In my opinion, just now it's not necessary to remove.  You can just error out if failed and continue when created succeed.

**Comment by Mashimiao** [comment]
carried in #285

**Pull request closed without merging**